daemons/gptp/linux/src/linux_hal_common.cpp

```diff
@@ -213,6 +213,9 @@ void *LinuxTimerQueueHandler( void *arg ) {
 		    struct LinuxTimerQueueActionArg *arg = iter->second;
 			timerq->timerQueueMap.erase(iter);
 			timerq->LinuxTimerQueueAction( arg );
+			if( arg->rm ) {
+				delete arg->inner_arg;
+			}
 			timer_delete(arg->timer_handle);
 			delete arg;
```

Comment:
No it doesn't (I don't understand why you've asked me this twice now).  There's a huge memory leak we've confirmed the problem and the fix.  Thanks.